ushahidi/platform-client
# disabling apply-button when in map-view #2182

This pull request makes the following changes:
- disables the apply-button in filter-dropdown when in map-mode

Testing checklist:
- Go to data-view
- Open filter-dropdown
- [ ] Expected result: the apply-button should not be disabled
- Go to map-view
- Open the filter-dropdown
- [ ] Expected result: the apply-button should not be disabled
- Select a saved search
- Go to data-view
- Open filter-dropdown
- [ ] Expected result: the apply-button should not be disabled
- Go to map-view
- Open the filter-dropdown
- [ ] Expected result: the apply-button should not be disabled
- Select a collection
- Go to data-view
- Open filter-dropdown
- [ ] Expected result: the apply-button should not be disabled
- Go to map-view
- Open the filter-dropdown
- [ ] Expected result: the apply-button should not be disabled

- [x] I certify that I ran my checklist, checklist is here: https://ushahidi.ontestpad.com/script/6#1//

Fixes ushahidi/platform#2182 .

Ping @ushahidi/platform
